src/Microsoft.Security.Utilities.Benchmarks/RE2RegexEngine.cs

```diff
@@ -35,6 +31,28 @@ public IEnumerable<UniversalMatch> Matches(string input, string pattern, RegexOp
                     };
                 }
             }
+            else
+            {
+                if (Regex2.Matches(pattern, input, out List<Dictionary<string, FlexMatch>> matches, 256L * 1024L * 1024L))
```

Comment:
Use a named arg or constant here. I was wondering if this is a max length, but it's max memory.